AvalonStudio/AvalonStudio/SearchResultViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AvalonStudio.MVVM;
+using AvalonStudio.Projects;
+using AvalonStudio.Utils;
+using ReactiveUI;
+using System;
+using Avalonia.Media;
+using System.Collections.Generic;
+
+namespace AvalonStudio.Controls
+{
+    public class SearchResultViewModel : ViewModel<ISourceFile>, IComparable<SearchResultViewModel>
+    {
+        public SearchResultViewModel(ISourceFile model) : base(model)
+        {
+        }
+
+        public int Priority { get; set; }
+
+        public string Title => Model.Name;
+
+        public string Location => $"{Model.Project.Name}\\{Model.Project.Location.MakeRelativePath(Model.Location)}";
```

Comment:
Concatenating the project name here indicates it wasn't searching against the project name, do the same in your query code.